adb-logcat: add page (#8424)
tldr-pages/tldr: pages/common/adb-logcat.md

> # logcat

Reviewer: ```suggestion
# adb-logcat
```
Even though it is an extension of `logcat` , I think mentioning `adb-logcat` would be better

> Dump a log of system messages.
> More information: <https://developer.android.com/studio/command-line/logcat>.

- Display system logs:

`adb logcat`

- Display lines that match a regular expression:

`adb logcat -e {{regular_expression}}`

> - Filter the log with filterspecs(`{{complete_tag:[V | D | I | W | E | F | S]}}`), `*` for all but not a wildcard(`*:S` suppresses others)

Reviewer 2: I think we should change this as well. Its quite confusing.

> - Filter the log with filterspecs(`{{complete_tag:[V | D | I | W | E | F | S]}}`), `*` for all but not a wildcard(`*:S` suppresses others):
> 
> `adb logcat {{tag0:V}} {{tag1:D}} *:S`

Reviewer 3: I think we can split this into 3 examples to be easier to understand
```suggestion
- Display logs for a tag in a specific mode ([V]erbose, [D]ebug, [I]nfo, [W]arning, [E]rror, [F]atal, [S]ilent), filtering other tags:

`adb logcat {{tag}}:{{mode}} *:S`

- Display logs for React Native applications in [V]erbose mode [S]ilencing other tags:

`adb logcat ReactNative:V ReactNativeJS:V *:S`

- Display logs for all tags with priority level [W]arning and higher:

`adb logcat *:W`
```

> - Color the log(usually use with filters)

Reviewer: ```suggestion
- Color the log(usually use with filters):
```

# adb-logcat

> Dump a log of system messages.
> More information: <https://developer.android.com/studio/command-line/logcat>.

- Display system logs:

`adb logcat`

- Display lines that match a regular expression:

`adb logcat -e {{regular_expression}}`

- Display logs for a tag in a specific mode ([V]erbose, [D]ebug, [I]nfo, [W]arning, [E]rror, [F]atal, [S]ilent), filtering other tags:

`adb logcat {{tag}}:{{mode}} *:S`

- Display logs for React Native applications in [V]erbose mode [S]ilencing other tags:

`adb logcat ReactNative:V ReactNativeJS:V *:S`

- Display logs for all tags with priority level [W]arning and higher:

`adb logcat *:W`

> - Color the log(usually use with filters):

Reviewer 2: ```suggestion
- Color the log (usually use with filters):
```

Reviewer 3: I'll apply this tiny fix and merge it.
